packages/frontend-plugin-api/api-report.md

```diff
@@ -662,16 +662,17 @@ export type CreateExtensionBlueprintOptions<
   },
 > = {
   kind: string;
-  namespace?: string;
+  namespace?: string | ((params: TParams) => string);
   attachTo: {
     id: string;
     input: string;
   };
   disabled?: boolean;
   inputs?: TInputs;
   output: Array<UOutput>;
+  name?: string | ((params: TParams) => string);
```

Comment:
I added name here, It's gonna be needed to do the `createThemeExtension` https://github.com/backstage/backstage/blob/31bfc4417f7a4d04d5899ce1d638505c30f9f627/packages/frontend-plugin-api/src/extensions/createThemeExtension.ts#L25